protocol/0081-SUCM-successor_markets.md

```diff
@@ -70,7 +71,32 @@ Two proposals that name the same parent can be submitted. Both can be approved b
 
 A new market proposal sets parent market Id to a market that has settled. The parent market has non-zero insurance pool balance. If the new market clears the opening auction before `parent settlement time + market.value.windowLength` then the virtual stakes are carried over and the relevant fraction of the insurance pool is transferred over (<a name="0081-SUCM-006" href="#0081-SUCM-006">0081-SUCM-006</a>).
 
-A new market proposal sets parent market Id to a market that has settled. The parent market has non-zero insurance pool balance. If the new market clears the opening auction after `parent settlement time + market.value.windowLength` then no virtual stakes are carried over, there is no transfer into the insurance pool of the new market from the parent and the new market has no parent market Id set (<a name="0081-SUCM-007" href="#0081-SUCM-007">0081-SUCM-007</a>).
+A new market proposal sets parent market Id to a market that has settled. The parent market has non-zero insurance pool balance. If the new market clears the opening auction after `parent settlement time + market.value.windowLength` then no virtual stakes are carried over, there is no transfer into the insurance pool of the new market from the parent and the new market has no parent market Id set (<a name="0081-SUCM-007" href="#0081-SUCM-007">0081-SUCM-007</a>)
+
+Successor markets can't be enacted if the parent market is still in the "proposed" state (<a name="0081-SUCM-008" href="#0081-SUCM-008">0081-SUCM-008</a>)
```

Comment:
I would like this to be made very clear @davidsiska-vega, because IIRC, there shouldn't be anything to prevent a successor market from being enacted/leaving opening auction here, it's just that it would be a rather pointless thing to do, because there simply would be no ELS to carry over from the parent market.

The question here would then be: should the successor market be rejected, or should we clear the parent ID (ie enact this successor as a separate market)?

Reply:
This AC isn't correct and should be removed from the PR.